README-source.md

```diff
@@ -467,6 +467,8 @@ In some cases, it might be necessary to provide fields that are dynamically gene
 
 > You should see `bundle.inputData` partially filled in as users provide data - even in field retrieval. This allows you to build hierarchical relationships into fields (EG: only show issues from the previously selected project).
 
+> A function that returns dynamic fields cannot return additional functions to call for dynamic fields.
```

Comment:
minor tweak that helped my visualize what this was saying "A function that returns a list of dynamic fields cannot include additional functions in that list to call for dynamic fields". take it or leave it